### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by Killian264**

### CODE REVIEW

**Comment by Killian264** [comment]
Set requirements on passwords. 

Have some way of checking duplicate emails.

File: golocker/services/user/user.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+	if err != nil {
+		return err
+	}
+
+	err = judgePasswordStrength(user.Password)
+
+	if err != nil {
+		return err
+	}
```
Comment (Killian264) [inline golocker/services/user/user.go:46]: move validation into seperate funciton

Comment (Killian264) [inline golocker/services/user/user.go:46]: move into validation funciton

Comment (CastleAnzalone) [reply golocker/services/user/user.go]: Solved in last commit

**Pull request merged**